collections/brandedColleges.js

```diff
@@ -8,6 +8,7 @@ BrandedColleges.attachSchema(new SimpleSchema({
   aiSubject: {type: String},
   animal: {type: String, optional: true}, // Panther
   askInternet: {type: Boolean, optional: true}, // AH-352
+  availableRegions: {type: [String], optional: true},
```

Comment:
Could we make this non-optional and default to []?

Reply:
Just for reference, it's optional since only one partner is using the functionality.

Reply:
I don't mind either way. I can set it to be a default to []